### PR TITLE
Feat/pn 2394 separare pagine informativa

### PR DESCRIPTION
**Pull request by LorenzoCocciarelli**
## Short description
Both in pn-pa-webapp and pn-personafisica-webapp, the terms of service and privacy policy are now in two separate pages

## List of changes proposed in this pull request
- The terms of service and privacy policy are now in two separate pages, TermsOfService.page.tsx and PrivacyPolicy.page.tsx respectively
- Renamed old TermsOfService page to ToSAcceptance page

## How to test
- Check that terms of service and privacy policy are on 2 different routes and pages('informativa-privacy', 'termini di servizio') in pn-pa-webapp and pn-personafisica-webapp
- Check that links in the footer point to the respective pages.

### CODE REVIEW

File: packages/pn-pa-webapp/src/pages/PrivacyPolicy.page.tsx [hunk 1]
```diff
@@ -0,0 +1,389 @@
+import {
```
Comment (AndreaCimini90) [inline packages/pn-pa-webapp/src/pages/PrivacyPolicy.page.tsx:1]: Check if the page must be localized

File: packages/pn-pa-webapp/src/pages/TermsOfService.page.tsx [hunk 2]
```diff
@@ -1,53 +1,168 @@
...
+  Box, Container,
+  Link,
+  Typography
+} from "@mui/material";
 
-import { useAppDispatch, useAppSelector } from '../redux/hooks';
-import { acceptToS } from '../redux/auth/actions';
-import * as routes from '../navigation/routes.const';
-import { RootState } from '../redux/store';
+const PrivacyPolicyPage = () => (
```
Comment (AndreaCimini90) [inline packages/pn-pa-webapp/src/pages/TermsOfService.page.tsx:7]: Check if the page must be localized

File: packages/pn-personafisica-webapp/src/pages/PrivacyPolicy.page.tsx [hunk 3]
```diff
@@ -0,0 +1,403 @@
+import {
+  Box,
+  Container,
+  Link,
+  Typography
+} from "@mui/material";
+
+const PrivacyPolicyPage = () => (
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/pages/PrivacyPolicy.page.tsx:8]: Check if the page must be localized

File: packages/pn-personafisica-webapp/src/pages/TermsOfService.page.tsx [hunk 4]
```diff
@@ -1,52 +1,364 @@
...
+  Container,
+  Link,
+  Typography
+} from "@mui/material";
 
-import { useAppDispatch, useAppSelector } from '../redux/hooks';
-import { acceptToS } from '../redux/auth/actions';
-import * as routes from '../navigation/routes.const';
-import { RootState } from '../redux/store';
+const PrivacyPolicyPage = () => (
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/pages/TermsOfService.page.tsx:8]: Check if the page must be localized

**Review by AndreaCimini90 — COMMENTED** [review]
If the pages must be localized, I suggest to use Trans component in the way explained here https://react.i18next.com/latest/trans-component

File: packages/pn-pa-webapp/src/pages/TermsOfService.page.tsx (hunk 2 above)
Comment (LorenzoCocciarelli) [reply packages/pn-pa-webapp/src/pages/TermsOfService.page.tsx:7]: The terms of service and privacy policy pages don't need to be localized, as they are placeholder text and will be injected through a script in the future and managed through a cms by the legal team, as far as I know.

**Pull request merged**